staging/src/k8s.io/kubectl/pkg/metricsutil/metrics_printer.go

```diff
@@ -45,11 +40,31 @@ type ResourceMetricsInfo struct {
 }
 
 type TopCmdPrinter struct {
-	out io.Writer
+	out               io.Writer
+	measuredResources []v1.ResourceName
+	nodeColumns       []string
+	podColumns        []string
 }
 
-func NewTopCmdPrinter(out io.Writer) *TopCmdPrinter {
-	return &TopCmdPrinter{out: out}
+func NewTopCmdPrinter(out io.Writer, showSwap bool) *TopCmdPrinter {
+	printer := &TopCmdPrinter{
+		out: out,
+		measuredResources: []v1.ResourceName{
+			v1.ResourceCPU,
+			v1.ResourceMemory,
+		},
+		nodeColumns: []string{"NAME", "CPU(cores)", "CPU(%)", "MEMORY(bytes)", "MEMORY(%)"},
+		podColumns:  []string{"NAME", "CPU(cores)", "MEMORY(bytes)"},
+	}
+
+	if showSwap {
+		printer.measuredResources = append(printer.measuredResources, "swap")
```

Comment:
Can you make `swap` a constant, and use that instead of direct string.

Reply:
done